indexd/alias/drivers/alchemy.py

```diff
@@ -83,9 +86,16 @@ def __init__(self, conn, **config):
         
         Base.metadata.bind = self.engine
         Base.metadata.create_all()
-        
         self.Session = sessionmaker(bind=self.engine)
 
+        md = MetaData()
```

Comment:
can we make a separate migration function and script instead of check then conditionally migrate for driver initialization? This also doesn't work on sqlite mode.